TrashMob/client-app/src/components/Pages/Home.tsx

```diff
@@ -84,12 +93,22 @@ const Home: FC<HomeProps> = ({ isUserLoaded, currentUser, history, location, mat
                                 <h4 className='text-center text-sm-left mb-4 ml-sm-5 font-weight-bold banner-heading ml-sm-4'>
                                     Meet up. Clean up. Feel good.
                                 </h4>
-                                <Link
-                                    className='mb-3 mb-sm-4 btn btn-primary mx-auto ml-sm-5 banner-button mr-sm-auto ml-sm-4'
-                                    to='/gettingstarted'
-                                >
-                                    Join us today
-                                </Link>
+                                <div className='d-flex'>
+                                    {' '}
+                                    <Link
+                                        className='mb-3 mb-sm-4 btn btn-primary mx-auto ml-sm-5 banner-button mr-sm-auto ml-sm-4'
+                                        to='/gettingstarted'
+                                    >
+                                        Join us today
+                                    </Link>
+                                    <Button
```

Comment:
This is minor.

Hover color are different between Join us today button and Invite a friend button.

<img width="371" alt="image" src="https://github.com/user-attachments/assets/4ca79d8d-04a8-4f6e-ab59-9873de321fc3">

Reply:
Meh, some buttons behave that way, not worth putting the effort imo since we're migrating away from bootstrap